test/e2e/launch.js

```diff
@@ -464,10 +307,20 @@ function reduceTimeouts(nodeHome) {
     .split(`\n`)
     .map(line => {
       let [key, value] = line.split(` = `)
-      if (timeouts.indexOf(key) !== -1) {
-        return `${key} = ${parseInt(value) / 10}`
+
+      if (timeouts.indexOf(key) === -1) {
+        return line
       }
-      return line
+
+      // timeouts are in the format "100ms" or "5s"
+      value = value.replace(/"/g, ``)
+      if (value.trim().endsWith(`ms`)) {
+        value = parseInt(value.trim().substr(0, value.length - 2))
+      } else if (value.trim().endsWith(`s`)) {
+        value = parseInt(value.trim().substr(0, value.length - 1)) * 1000
+      }
```

Comment:
It would be good to put this parsing code into a separate function and write unit tests for it.